base/regex.jl

```diff
@@ -141,20 +141,18 @@ function getindex(m::RegexMatch, name::Symbol)
 end
 getindex(m::RegexMatch, name::AbstractString) = m[Symbol(name)]
 
-function contains(s::AbstractString, r::Regex, offset::Integer=0)
+function isfound(r::Regex, s::AbstractString; offset::Integer=0)
```

Comment:
Shouldn't `offset` be a positional argument just like the index passed to `findnext`? Then there could be a generic definition `isfound(needle, haystack, i) = findnext(needle, haystack, i) !== nothing`.

Another possibility would be to have keyword arguments like `start` and `stop`, which would allow specifying the ending index without passing the starting index when needed.